docs/en/configure-elastic-serverless-forwarder.asciidoc

```diff
@@ -0,0 +1,265 @@
+:aws: AWS
```

Comment:
Is this an old file that needs to be deleted? It seems like an older version of the content in aws-elastic-serverless-forwarder-configuration.asciidoc. (This file does not exist in the observability-docs repo.)

Reply:
I've removed this file and  aws-serverless-transform.asciidoc because they are not used and seem to contain old content. Incidentally, I'm not sure why the content under Transform AWS data was moved to a topic called Configuration options. The new title is too generic and and the topic mixes things that I don't think really belong together. Anyhow, I can revisit this later. Main goal here is to get the content merged and more or less looking like it did in the obs guide.